Cargo.toml

```diff
@@ -1,9 +1,11 @@
 [workspace]
 members = [
     "rust/template",
+    "rust/template/ovsdb",
     "rust/template/cmd_parser",
-    "rust/template/differential_datalog",
+    "rust/template/ddlog_derive",
```

Comment:
Why does this have to be a separate crate?

Reply:
Procedural macros have to be separate crates (that can only export proc/derive macros) since they're compiled and ran at compile time